aries_cloudagent/storage/askar.py

```diff
@@ -263,8 +263,12 @@ def __init__(
         self.type_filter = type_filter
         self.page_size = page_size or DEFAULT_PAGE_SIZE
         self._done = False
-        self._scan = None
         self._profile = profile
+        self._scan = self._profile.store.scan(
+            self.type_filter,
+            self.tag_query,
+            profile=self._profile.settings.get("wallet.id")
+        )
```

Comment:
self._scan should stay None in the constructor. It is set later in open method

Reply:
Fixed